.travis.yml

```diff
@@ -1,14 +1,13 @@
 # Travis CI configuration file for running tests
 language: python
+sudo: false
 python:
   - "2.7"
-before_install:
-  - pip install pep8
-  - pep8 --verbose
 install:
   - pip install --allow-all-external -r requirements.txt
-  - pip install coverage
+  - pip install --allow-all-external -r test_requirements.txt
 script:
+  - pep8 --verbose
```

Comment:
Is this change making it so that the tests no longer "fail fast" on PEP8 errors?